src/trx/game/output/sources/objects.c

```diff
@@ -224,13 +224,9 @@ static void M_Stage(const OBJECT_MESH *const mesh, const bool skybox)
         },
         .room = Output_GetCurrentRoom(),
     };
-    if (skybox) {
```

Comment:
The skybox argument can likely be removed now?

Reply:
Updated 😌